content/sv/codebase.md

```diff
@@ -0,0 +1,17 @@
+## I. Codebase (Kodbas)
+### En kodbas med versionshantering, flera produktionssättningar
+
+En tolvfaktorsapplikation hålls alltid i någon form av versionskontrollsystem, som exempelvis [Git](http://git-scm.com/), [Mercurial](https://www.mercurial-scm.org/), eller [Subversion](http://subversion.apache.org/). En kopia av versionshanteringsdatabasen kallas för ett *kodrepository*, ofta förkortat till *kodrepo* eller just *repo*.
+
+En *kodbas* är varje enskilt repo (i ett centraliserat versionshanteringssystem såsom Subversion), eller varje set av repon som delar en viss koduppladdning som bas (i ett decentraliserat versionshanteringssystem såsom Git).
+
+![En kodbas kopplas till flera produktionssättningar](/images/codebase-deploys.png)
+
+Det finns alltid ett ett-till-ett-förhållande mellan kodbasen och applikationen:
+
+* Om det finns multipla kodbaser, är det inte en applikation -- det är ett distribuerat system. Varje komponent i ett distribuerat system är sin egen applikation, och kan således i sin egen rätt följa tolvfaktor-modellen.
+* Multipla applikationer som delar samma kod bryter mot tolvfaktor-modellen. Lösningen är att flytta ut delad kod till bibliotek som kan inkluderas genom [beroenden](./dependencies).
+
+Det finns alltid bara en kodbas per applikation, men det kommer bli många produktionssättningar av samma applikation. En *produktionssättning* är en körbar instans av applikationen. Det kan exempelvis vara en produktionssajt, eller en eller flera sajter för acceptanstestning. Dessutom har ofta respektive utvecklare en kopia av applikationen som kan köras lokalt i den egna utvecklingsmiljön, vilket också räknas som en produktionssättning.
```

Comment:
```suggestion
Det finns alltid bara en kodbas per applikation, men det kommer bli många produktionssättningar av samma applikation. En *produktionssättning* är en körbar instans av applikationen. Det kan exempelvis vara en produktionssajt, eller en eller flera sajter för testning. Dessutom har ofta respektive utvecklare en kopia av applikationen som kan köras lokalt i den egna utvecklingsmiljön, vilket också räknas som en form av produktionssättning.
```

I don't think we need to specifically mention acceptance testing. There is no good word matching "staging", in swedish speech the word staging is used but is not a swedish word. Produktionssättning roughly translates to production placement and is the word used when deploying to production. More general words used in swedish are "release" and "deploy", possibly "utrullning"(roll out). Therefore I suggest adding "en form av produktionssättning", "a shape/type of production deploy".